crates/ruff_python_parser/src/lexer.rs

```diff
@@ -1455,12 +1458,12 @@ impl StringFlags for TokenFlags {
 impl TokenFlags {
     /// Returns `true` if the token is an f-string.
     const fn is_f_string(self) -> bool {
-        self.contains(TokenFlags::F_STRING)
+        self.intersects(TokenFlags::F_STRING)
     }
 
     /// Returns `true` if the token is a raw string.
     const fn is_raw_string(self) -> bool {
-        self.contains(TokenFlags::RAW_STRING)
+        self.intersects(TokenFlags::RAW_STRING)
     }
```

Comment:
Using `contains` is incorrect here.

For all other changes, I find `intersects` easier to reason about.